demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Logger, LogLevel } from '../../../../src/index';
+
+export default class CloudWatchLogger implements Logger {
```

Comment:
Sorry, I did not catch this in your previous PR, so I just force-pushed a fix on top of your commit. Please pull the latest change from the remote ```cwl```.

I don't see a good reason to change the implementation detail ```ConsoleLogger.log()``` to ```protected``` and extended ```ConsoleLogger``` because we did not use the ConsoleLogger's feature at all but duplicate writing console logging code.

In this PR, I updated two:
- Implement ```Logger``` instead of extending ```ConsoleLogger```.
- Revert ```ConsoleLogger``` back. Now, this PR does not contain  ```ConsoleLogger.ts```, ```consolelogger.html```, and ```search.js```.